normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
```

Comment:
Also, you should do this as

```python
with open('normandy/recipes/static/actions/console-log/package.json') as f:
    schema = json.load(f)
```

This way the file is automatically closed, and also `json.load` will read directly from files.